### PR TITLE
fix google analytics

### PR DESCRIPTION
**Pull request by toddmedema**

### CODE REVIEW

File: app/Main.tsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ import {openSnackbar} from './actions/Snackbar'
 import {silentLogin} from './actions/User'
 import {handleRemotePlayEvent} from './actions/RemotePlay'
 import {getStore} from './Store'
-import {getWindow, getGapi, getGA, getDevicePlatform, getDocument, setGA, setupPolyfills} from './Globals'
+import {getAppVersion, getWindow, getGapi, getGA, getDevicePlatform, getDocument, setGA, setupPolyfills} from './Globals'
```
Comment (smartin015) [inline app/Main.tsx]: Why not just import a constant value from `./Constants`?

File: app/Main.tsx [hunk 2]
```diff
@@ -138,7 +150,7 @@ function setupEventLogging() {
     });
   } else {
     window.FirebasePlugin = {
-      logEvent: (name: string, args: any) => { console.log(name, args); },
+      logEvent: (name: string, args: any) => { console.info('Firebase log: ', name, args); },
```
Comment (smartin015) [inline app/Main.tsx]: If this is done *instead* of firebase logging, we should probably indicate so in the console print.

File: app/Main.tsx [hunk 3]
```diff
@@ -152,35 +164,19 @@ function setupHotReload() {
   }
 }
 
+// disabled during local dev
```
Comment (smartin015) [inline app/Main.tsx]: Can we avoid initialization entirely when we're running in dev mode?

File: app/Main.tsx (hunk 1 above)
Comment (toddmedema) [reply app/Main.tsx]: `getAppVersion()` is used in a number of places; it pulls from package.json, and prevents us from having to define the app version in two places. I suppose we could also move getAppVersion to an APP_VERSION constant that equals `PACKAGE.version`, though I suspect that might break tests / might not be worth the hassle

File: app/Main.tsx (hunk 2 above)
Comment (toddmedema) [reply app/Main.tsx]: Good call

File: app/Main.tsx (hunk 3 above)
Comment (toddmedema) [reply app/Main.tsx]: Fair; had to undisable it for testing, but should return to being totally disabled. Fixed.

**Pull request merged**